docs/content/getting-started/hello-dagster.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Hello Dagster | Dagster
+description: Run dagster for the first time
+---
+
+# Hello Dagster
+
+This guide helps you build a simple data pipeline in dagster that downloads the top 10 HackerNews stories.
+
+Start by creating a file called `hello-dagster.py` and copying the following:
+
+```python file=/getting-started/hello-dagster/hello-dagster.py
+import pandas as pd
+import requests
+
+from dagster import MetadataValue, Output, asset, repository
+
+
+@asset
+def hackernews_top_story_ids():
+    """
+    Get top stories from the HackerNews top stories endpoint.
+    API Docs: https://github.com/HackerNews/API#new-top-and-best-stories
+    """
+    top_story_ids = requests.get(
+        "https://hacker-news.firebaseio.com/v0/topstories.json"
+    ).json()
+    return top_story_ids[:10]
+
+
+# asset dependencies can be inferred from inputs
+@asset
+def hackernews_top_stories(hackernews_top_story_ids):
+    """Get items based on story ids from the HackerNews items endpoint"""
+
+    results = []
+    for item_id in hackernews_top_story_ids:
+        item = requests.get(
+            f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        ).json()
+        results.append(item)
+
+    df = pd.DataFrame(results)
+
+    # recorded metadata can be customized
+    metadata = {
+        "num_records": len(df),
+        "preview": MetadataValue.md(df.to_markdown()),
+    }
+
+    return Output(value=df, metadata=metadata)
+
+
+@repository
+def hello_dagster():
+    return [hackernews_top_story_ids, hackernews_top_stories]
+```
+
+Now install the Python packages we'll need to run `hello-dagster.py` in your favorite Python environment. Unsure? Follow our [installation guide](/getting-started/install).
+
+```bash
+# run in a terminal in your favorite python environment
+pip install dagster dagit pandas
+```
+
+We'll use `dagit`, the dagster user interface, to run `hello-dagster.py`:
```

Comment:
A few things here:

```suggestion
Using Dagit, the Dagster user interface, run `hello-dagster.py`:
```

- Dagit and Dagster should be capitalized. I think here you were referring to the command to start Dagit, but the sentence reads like it's actually the name / proper noun.

   You could say something like "Next, run `dagit` in your terminal to start Dagit, Dagster's user interface" and that would work.
- The instructions should clarify that the user needs to run this command in the same location as `hello-dagster.py`. I think it's implied, but given the number of issues beginners have I think being overly specific is fine / preferred